spec/features/answers/answers_have_comments_spec.rb

```diff
@@ -1,10 +1,16 @@
 require 'rails_helper'
 
 feature 'user views a question' do
+  attr_reader :user, :question, :answer
+  before(:each) do
+    @user = Fabricate(:user, password: 'password')
+    @user.roles.create(name: 'registered_user')
```

Comment:
I think we can use Fabricator to make UserRoles, and it might be more consistent.